staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4160,15 +4160,24 @@ const (
 	ServiceInternalTrafficPolicyLocal ServiceInternalTrafficPolicyType = "Local"
 )
 
-// Service External Traffic Policy Type string
+// ServiceExternalTrafficPolicy describes the type of traffic routing for
+// external traffic.
 // +enum
-type ServiceExternalTrafficPolicyType string
+type ServiceExternalTrafficPolicy string
 
 const (
-	// ServiceExternalTrafficPolicyTypeLocal specifies node-local endpoints behavior.
-	ServiceExternalTrafficPolicyTypeLocal ServiceExternalTrafficPolicyType = "Local"
-	// ServiceExternalTrafficPolicyTypeCluster specifies node-global (legacy) behavior.
-	ServiceExternalTrafficPolicyTypeCluster ServiceExternalTrafficPolicyType = "Cluster"
+	// ServiceExternalTrafficPolicyLocal specifies node-local endpoints behavior.
+	ServiceExternalTrafficPolicyLocal ServiceExternalTrafficPolicy = "Local"
+	// ServiceExternalTrafficPolicyCluster specifies node-global (legacy) behavior.
+	ServiceExternalTrafficPolicyCluster ServiceExternalTrafficPolicy = "Cluster"
+)
+
+// for backwards compat
+type ServiceExternalTrafficPolicyType = ServiceExternalTrafficPolicy
```

Comment:
```go
// for backwards compat
// +enum
type ServiceExternalTrafficPolicyType = ServiceExternalTrafficPolicy
```
To workaround a k/gengo bug.

Reply:
Why does `IPFamily` not show up in swagger.json (field name IPFamilies) as an enum?

Reply:
I did as you suggest but it doesn't really work:

It half works for 1 case, but duplicates the enum values in the list, and doesn't work at all for the other cases.